powerdns_recursor/metadata.csv

```diff
@@ -55,7 +55,6 @@ powerdns.recursor.auth6_answers1_10,gauge,,query,second,Number of queries per se
 powerdns.recursor.auth6_answers10_100,gauge,,query,second,Number of queries per second answered by auth6s within 100 milliseconds; Available since pdns_recursor v4.x,0,powerdns_recursor,auth6s answers/sec < 100ms
 powerdns.recursor.auth6_answers100_1000,gauge,,query,second,Number of queries per second answered by auth6s within 1 second; Available since pdns_recursor v4.x,0,powerdns_recursor,auth6s answers/sec < 1s
 powerdns.recursor.dlg_only_drops,gauge,,record,second,The number of records dropped per second because of 'delegation only' setting; Available since pdns_recursor v4.x,0,powerdns_recursor,dlg only drops
-powerdns.recursor.dlg_only_drops,gauge,,record,second,The number of records dropped per second because of 'delegation only' setting; Available since pdns_recursor v4.x,0,powerdns_recursor,dlg only drops
```

Comment:
duplicate?

Reply:
complete duplicate

Reply:
That was obvious actually. I only looked at the Github diff. Thanks for clarifying tho.